bitnami/influxdb/templates/application.yaml

```diff
@@ -0,0 +1,323 @@
+              value: {{ .Values.bucket | quote }}
+            {{- end }}
+            {{- if eq .Values.objectStore "s3" }}
+            - name: AWS_DEFAULT_REGION
+              value: {{ .Values.s3.defaultRegion | quote }}
+            {{- if .Values.s3.endpoint }}
+            - name: AWS_ENDPOINT
+              value: {{ .Values.s3.endpoint | quote }}
+            {{- end }}
+            {{- if .Values.usePasswordFiles }}
+            - name: AWS_ACCESS_KEY_ID_FILE
+              value: "/opt/bitnami/influxdb/secrets/s3-access-key-id"
+            - name: AWS_SECRET_ACCESS_KEY_FILE
+              value: "/opt/bitnami/influxdb/secrets/s3-secret-access-key"
+            {{- else }}
+            - name: AWS_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "influxdb.store.secret.name" . }}
+                  key: s3-access-key-id
+            - name: AWS_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "influxdb.store.secret.name" . }}
+                  key: s3-secret-access-key
+            {{- end }}
+            {{- else if eq .Values.objectStore "google" }}
+            {{- if .Values.usePasswordFiles }}
+            - name: GOOGLE_SERVICE_ACCOUNT_FILE
+              value: "/opt/bitnami/influxdb/secrets/google-service-account-key"
+            {{- else }}
+            - name: GOOGLE_SERVICE_ACCOUNT
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "influxdb.store.secret.name" . }}
+                  key: google-service-account-key
+            {{- end }}
+            {{- else if eq .Values.objectStore "azure" }}
+            - name: AZURE_STORAGE_ACCOUNT
+              value: {{ .Values.azure.account | quote }}
+            {{- if .Values.usePasswordFiles }}
+            - name: AZURE_STORAGE_ACCESS_KEY_FILE
+              value: "/opt/bitnami/influxdb/secrets/azure-storage-access-key"
+            {{- else }}
+            - name: AZURE_STORAGE_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "influxdb.store.secret.name" . }}
+                  key: azure-storage-access-key
+            {{- end }}
+            {{- end }}
+            {{- if .Values.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.extraEnvVars "context" .) | nindent 12 }}
+            {{- end }}
+          {{- if or .Values.extraEnvVarsCM .Values.extraEnvVarsSecret }}
+          envFrom:
+            {{- if .Values.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ .Values.extraEnvVarsCM }}
+            {{- end }}
+            {{- if .Values.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ .Values.extraEnvVarsSecret }}
+            {{- end }}
+          {{- end }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.containerPorts.http }}
```

Comment:
I don't see the port configured in the env vars. Is it configurable or hardcoded?

Reply:
I see it can be configured 

https://github.com/influxdata/influxdb/blob/bf83e7fbb397e579cbee0ca87c5b797fc2ecdcf5/influxdb3/src/help/serve.txt#L26

Reply:
addressed at https://github.com/bitnami/charts/pull/33743/commits/859c509824da5be460e8f02a9701f49e440ae834